src/components/application_manager/src/resumption/resumption_data_processor.cc

```diff
@@ -858,4 +912,64 @@ void ResumptionDataProcessor::CheckVehicleDataResponse(
   }
 }
 
+void ResumptionDataProcessor::CheckCreateWindowResponse(
+    const smart_objects::SmartObject& request,
+    const smart_objects::SmartObject& response) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  const auto correlation_id =
+      response[strings::params][strings::correlation_id].asInt();
+
+  const auto& msg_params = request[strings::msg_params];
+  const auto app_id = msg_params[strings::app_id].asInt();
+
+  auto application = application_manager_.application(app_id);
+  if (!application) {
+    LOG4CXX_ERROR(logger_,
+                  "Application is not registered by hmi id: " << app_id);
+    return;
+  }
+
+  const auto window_id = msg_params[strings::window_id].asInt();
+  if (!IsRequestSuccessful(response)) {
```

Comment:
`IsRequestSuccessful(response) ` confusing. IsRequestSuccessful probably should receive request as parameter

Reply:
@LuxoftAKutsan as I can see it actually verifies responses, not requests. I renamed it accordingly.
See https://github.com/LuxoftSDL/sdl_core/pull/70/commits/6b8a138a35e6abea6bf9b6841a0d21d96c2bc431